### PR TITLE
Plugin Management: implement remove plugin action for installed sites list on plugin details page

### PR DESCRIPTION
**Pull request by yashwin**
#### Proposed Changes

This PR adds an action to remove the plugin for the installed sites list on the plugin details page.

#### Testing Instructions

**Prerequisites**

Since this change is made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

**Instructions**

1. Run `git checkout add/plugin-details-actions-for-installed-on-sites` and `yarn start-jetpack-cloud`
2. Open http://jetpack.cloud.localhost:3000/, and you'll be redirected to the /dashboard.
3. Click on **Plugins** in the sidebar -> Click on any plugin -> Verify that action to remove the plugin is being shown for all the installed sites

Screenshot 2022-08-25 at 6 10 55 PM

4. Click on `Remove` and verify that the plugin is removed and appears on the **Available on** list. 
5. Try removing a plugin from Calypso Blue and verify that it works as expected.

> **_NOTE:_** Actions for plugins in multi-site and single-site views will be added in upcoming PRs

#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into the trunk. Inapplicable items can be left unchecked.

The PR author and reviewer are responsible for completing the checklist.

- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes? Will be added later - 1202518759611394-as-1202627702018877
- [x] ~~Have you tested the feature in [Simple](P9HQHe-k8-p2-p2), [Atomic](P9HQHe-jW-p2-p2), and [self-hosted Jetpack sites](PCYsg-g6b-p2-p2)?~~
- [x] Have you checked for TypeScript, React, or other console errors?
- [x] Are we memoizing when appropriate (for expensive computations)? More info in [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md) and [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors)
- [x] Have we sent any new strings [for translation](PCYsg-1vr-p2-p2) ASAP?

Related to 1202518759611394-as-1202698250096170

### CODE REVIEW

File: client/my-sites/plugins/plugin-management-v2/plugin-common/plugin-common-list/index.tsx [hunk 1]
```diff
@@ -13,7 +13,8 @@ interface Props {
 	items: Array< any >;
 	isLoading: boolean;
 	columns: Columns;
-	hasMoreActions: boolean;
+	title?: ReactNode;
```
Comment (vitozev) [inline client/my-sites/plugins/plugin-management-v2/plugin-common/plugin-common-list/index.tsx]: Maybe we need to change this to `header`?

**Review by vitozev — APPROVED** [review]
LGTM! The manual tests worked well, I didn't find any regression.

I left one comment, which is probably the reason why there are some failed tests.

**Comment by yashwin** [comment]
> LGTM! The manual tests worked well, I didn't find any regression.
> 
> I left one comment, which is probably the reason why there are some failed tests.

Thanks for the review, @vitozev. Yes, you are right. I have fixed it now

**Review by vitozev — APPROVED** [review]
🚢 

FWIW, in https://github.com/Automattic/wp-calypso/pull/66366 the `plugins/plugin-details-layout` feature flag was enabled only for development environments. This means that we won't display the "Installed plugins" table on the `/plugins/:slug/:site` route. See pdtkmj-kQ-p2 for more details.

**Pull request merged**